core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.module.ts

```diff
@@ -40,7 +41,8 @@ import { MergeSkillModalComponent } from 'components/skill-selector/merge-skill-
   imports: [
     BrowserModule,
     HttpClientModule,
-    SharedComponentsModule
+    SharedComponentsModule,
+    DynamicContentModule,
```

Comment:
`DynamicContentModule` is exported in `SharedComponentsModule`.  Why is it imported here?

Reply:
fixed.